src/objectid.ts

```diff
@@ -33,7 +33,7 @@ export class ObjectId {
   }
 
   /** @internal */
-  static index = Math.floor(Math.random() * 0xffffff);
+  private static index = Math.floor(Math.random() * 0xffffff);
```

Comment:
love the user of ts-private variables.  is this a new pattern we want to use?

Reply:
nvm it looks like we're already using it.  But i love it anyways

Reply:
Yea seems worth applying where it's relevant, this was internal already so hidden from users (with types) this hides it from the rest of the BSON library too